tests/ftests/081-pybindings-cgrp_get_curr_ctrl_path-v1.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# cgroup_get_current_controller_path() test using the python bindings (cgroup v1)
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Kamalesh Babulal <kamalesh.babulal@oracle.com>
+#
+
+from cgroup import Cgroup as CgroupCli, Mode
+from libcgroup import Cgroup, Version
+from process import Process
+import consts
+import ftests
+import sys
+import os
+
+
+CGNAME = '081getctrlpathv1'
+CONTROLLER = 'cpu'
+
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if config.args.container:
+        result = consts.TEST_SKIPPED
+        cause = 'This test cannot be run within a container'
+        return result, cause
+
+    if Cgroup.cgroup_mode() == Mode.CGROUP_MODE_UNIFIED:
+        result = consts.TEST_SKIPPED
+        cause = 'This test requires the legacy cgroup v1 hierarchy'
+
+    return result, cause
+
+
+def setup(config):
+    CgroupCli.create(config, CONTROLLER, CGNAME)
+
+    config.process.create_process_in_cgroup(config, CONTROLLER, CGNAME, ignore_systemd=True)
+
+
+def test(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    expected_path = os.path.join("/" + CGNAME)
```

Comment:
nitpick - I don't think `os.path.join()` is doing anything for us here since we're only providing a single string.